src/targets/gpu/fuse_mlir.cpp

```diff
@@ -314,6 +315,58 @@ void fuse_mlir::apply(module_pass_manager& mpm) const
 #endif
 }
 
+#ifdef MIGRAPHX_MLIR
+
+namespace {
+MIGRAPHX_PRED_MATCHER(is_supported_arch, instruction_ref)
+{
+    // TODO(ravil): debug
+    static std::unordered_set<std::string> supported_consumer_archs{
```

Comment:
This list isn't right.

Also, I'm pretty sure we're only meant to offload on `gfx110x`

Reply:
>This list isn't right.

You are correct. I just used it to test the concept on MI200 because I am still waiting to get access to a Navi machine.

From what I understood, `gfx11*` or `gfx110*` denote Navi3X GPUs. Maybe we can use a regex to capture current and future Navi cards?

Yi just updated me a few hours ago that gfx1100 for Navi31, gfx1101 for Navi32